Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcClient.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Blizzard.IO.Core.Rpc;
+using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public class NetqRabbitRpcClient : IRpcClient
+    {
+        private readonly INetqRabbitRpcConnection<Func<Type, object>> _netqRpcRabbitConnection;
+        private readonly ILogger _logger;
+
+        public NetqRabbitRpcClient(INetqRabbitRpcConnection<Func<Type, object>> netqRpcRabbitConnection, ILogger logger)
```

Comment:
You should get `LoggerFactory` instead of `ILogger`

Reply:
f